db/v-2/insert-test-data/sqlRequests/user.csv

```diff
@@ -0,0 +1,13 @@
+id;name;password;role;source;avatar;email;company;location;linkedin;git_hub;twitter
+1;"admin";"{bcrypt}$2a$12$7jB6.fJ/uHZiti94w0.QOu0M/lPzKU1lvX9TLL3uZHFjLGIfz1gHq";"ROLE_SUPER_ADMIN";"basic";"null";"null";"null";"null";"null";"null";"null"
+2;"user";"noop{123}";"ROLE_VIEWER";"basic";"null";"null";"null";"null";"null";"null";"null"
```

Comment:
You don't need to add _null_ here. It is enough not to write anything between semicolons 
2;"user";"noop{123}";"ROLE_VIEWER";"basic";;;;;;